packages/edit-site/src/components/sidebar-edit-mode/template-panel/hooks.js

```diff
@@ -51,9 +51,10 @@ function preparePatterns( patterns, template, currentThemeStylesheet ) {
 
 	return patterns
 		.filter(
-			filterOutExcludedPatternSources &&
-				filterOutDuplicatesByName &&
-				filterCompatiblePatterns
+			( pattern, index, items ) =>
+				filterOutExcludedPatternSources( pattern ) &&
+				filterOutDuplicatesByName( pattern, index, items ) &&
```

Comment:
I think any duplicates by name will be [removed here](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-block-patterns-registry.php#L118), so I wonder if we need this filter?

Reply:
Looks like we don't then :)

Reply:
Gone :-)

Reply:
This might warrant a deeper look as it's not the only place where duplicates were filtered out by name.